helm-charts/0.22.4/charts/sentinel/templates/Deployment.yaml

```diff
@@ -46,10 +46,13 @@ spec:
             - name: spire-agent-socket
               mountPath: /spire-agent-socket
               readOnly: true
-          {{- if .Values.initCommand.enabled }}
+          {{- if .Values.global.sentinelInitCommand.enabled }}
```

Comment:
@abhishek44sharma — I had to put it to .Values.global (instead of values.yml of sentinel) because TCX build pipeline was not able to read it otherwise.

I also thought about creating a separate helm chart for TCX, but that would be an overkill.

If we figure out to do it cleaner in TCX, we might refactor this part again.